models/poison_messages.go

```diff
@@ -40,6 +40,7 @@ type MessagePayload struct {
 	TimeSent time.Time `json:"time_sent" bson:"time_sent"`
 	Size     int       `json:"size" bson:"size"`
 	Data     string    `json:"data" bson:"data"`
+	Headers  string    `json:"headers"`
```

Comment:
where is the bson tag? poison messages are stored in the DB so they have to be stored with the headers